homeassistant/components/steam_online/config_flow.py

```diff
@@ -108,6 +108,11 @@ async def async_step_reauth_confirm(
         )
 
 
+def _batch_ids(ids: list[str]) -> Generator:
+    for i in range(0, len(ids), 275):
```

Comment:
What is the significance of 275? It would be helpful to have this stored in a descriptive constant or a comment describing its meaning.